### PR TITLE
Do one work unit, then stop

### PR DESCRIPTION
**Pull request by mlassnig**

### CODE REVIEW

**Comment by mlassnig** [comment]
we should probably stop the existing tasks on the _GPU queues and restart them with this

**Comment by lukasheinrich** [comment]
maybe we should do more than one? say 10? 20?

**Comment by mlassnig** [comment]
Yeah, why not. Do we have logs from MANC or MWT2 already with some timings? Should help us decide which value to put.

File: Dockerfile [hunk 1]
```diff
@@ -14,4 +14,4 @@ RUN dpkg -i /tmp/fahclient_7.5.1_amd64.deb || exit 0 && \
 WORKDIR /root
 VOLUME /root
 
-CMD ["FAHClient", "--user=CERN", "--team=38188", "--gpu=true", "--smp=true"]
+CMD ["FAHClient", "--user=CERN", "--team=38188", "--gpu=true", "--smp=true", "--max-units=1", "--exit-when-done"]
```
Comment (matthewfeickert) [inline Dockerfile:17]:
```suggestion
CMD ["FAHClient", "--user=CERN", "--team=38188", "--gpu=true", "--smp=true", "--max-units=20", "--exit-when-done"]
```

Comment (matthewfeickert) [reply Dockerfile:17]: @mlassnig If this is still an idea of interest can you updated the max units?

**Pull request closed without merging**